src/main/java/seedu/address/MainApp.java

```diff
@@ -83,22 +88,28 @@ public void init() throws Exception {
      */
     private Model initModelManager(Storage storage, UserPrefs userPrefs) {
         Optional<ReadOnlyAddressBook> addressBookOptional;
+        Optional<ReadOnlyUsersList> usersListOptional;
         ReadOnlyAddressBook initialData;
+        ReadOnlyUsersList initialUser;
         try {
             addressBookOptional = storage.readAddressBook();
+            usersListOptional = storage.readUsersList();
```

Comment:
You should put `readUsersList()` in a separate try-catch block. Otherwise the app wouldn't attempt to run `readUsersList()` after `readAddressBook()` fails.